components/styled/table/src/molecules/useDimensionRef.tsx

```diff
@@ -18,9 +21,11 @@ const useDimensionsRef = (elementRef: React.RefObject<HTMLElement>) => {
     }
 
     window.addEventListener('resize', handleResize)
+    window.addEventListener('updateDimensions', handleResize)
 
     return () => {
       window.removeEventListener('resize', handleResize)
+      window.removeEventListener('updateDimensions', handleResize)
```

Comment:
If this new event does exactly the same as the old one, why do we need to add a new one?

Reply:
Its triggered in the table-component.tsx file

Reply:
Yeah I saw, I was wondering if there's a reason we can't just trigger the `resize` event instead?

Reply:
we could all we need is to update dimensions if that's a better way to do it then yeah.